docs/release_notes/release_notes.md

```diff
@@ -14,7 +14,47 @@ For the full release notes, see the [Changelog](https://github.com/Katello/katel
+
+### OSTree Content
+
+Users can now manage OSTree content through Katello in the same ways that RPM content can be managed. This includes creating OSTree repositories, syncing those repositories and adding them to content views. Similar to RPM content, Atomic machines can be provisioned from local OSTree content. 
+
+### Lazy Sync
+
+### Docker V2
+
+### Mirror On Sync
+
+Repositories now have the option to be used in a "mirror on sync" state where the state of the repository mirrors that of the upstream repository at sync time. This results in packages no longer in the upstream repository being removed.
+
+### Remote Execution Integration
+
+It is possible now to use Foreman remote execution plugin to execute the content actions
+(install package, errata etc.). This acts as an alternative to the current katello-agent capabilities
+and allows to execute the actions over SSH (or other providers that will by the remote execution
+plugin later). It also allows to customize the script to be run when executing the content action
+as well as some advanced execution options (delayed run, recurring, splay time etc.).
+See http://theforeman.org/plugins/foreman_remote_execution/ for more details on the remote execution plugin.
+
+### Capsule Status on Proxy
+
+Proxy detail page was extended to provide statistics about synchronized content grouped by lifecycle environments, usage of pulp storage to warn users when the capsule is running out of disc space and information about synchronization status. The latter displays progress of current tasks when the synchronization is running. It also allows for starting and cancelling the synchronization process.
+
+The ability to cancel synchronizations as well as showing the content counts are now reflected in the cli. You can find it in commands hammer capsule content cancel-synchronization and hammer capsule content info. Please note that the command capsule has became an alias to proxy and they share the same subcommands.
+
+### Inter-Server Sync
+
+Users may now perform exports and imports of yum repositories to disk, either as a series of directories or as ISO files. This is especially useful for users in "air-gapped" network envionrments, where a Katello server might not have access to the Internet. Note that Inter-Server Sync (ISS) replaces the katello-disconnected tool. Additional information is available at (http://www.katello.org/docs/user_guide/disconnected/)[http://www.katello.org/docs/user_guide/disconnected/].
+
+### 
+
 ## Deprecation Warnings
 
+ * Katello 3.0 is a major update which means all deprecation warnings in 2.4 have been removed entirely.
+ * The katello-installer has been deprecated in favor of the foreman-installer with scenarios. Please see the installation instructions.
+
```

Comment:
We may want to add a note that support for Docker v1 has been fully deprecated and removed from Katello 3.0, due to the fundamental changes that come with Docker v2.  In addition, as part of the upgrade to Katello 3.0, any existing repositories containing Docker v1 content (images) will be removed.